tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -35,8 +35,6 @@ def tap_search_bar(self):
         search_bar.tap()
 
         # TODO These lines are a workaround for bug 1020974
-        import time
```

Comment:
FYI, There will be a conflict with Martijn's patch on this. He moved that function in System